app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/commontask/components/CommonTaskCreatedBy.kt

```diff
@@ -9,6 +9,7 @@ import io.eugenethedev.taigamobile.domain.entities.CommonTaskExtended
 import io.eugenethedev.taigamobile.domain.entities.User
 import io.eugenethedev.taigamobile.ui.components.lists.UserItem
 
+@Suppress("FunctionName")
```

Comment:
Why use it?

Reply:
Default linter rule is that non-composable function names shouldn’t start with upperase letter. So I placed it here (and in some other places) just to suppress the warning